### PR TITLE
Renamed data attribute

### PR DESCRIPTION
**Pull request by Dinnall**
### What's this PR do?

This pull request makes a change to the data attribute being called for testing in [PR](https://github.com/DoSomething/phoenix-next/pull/2151) to `data-testid` being used [here](https://github.com/DoSomething/communal-docs/tree/master/Contributing#tests) for testing.

### How should this be reviewed?

...

### Any background context you want to provide?

...

### Relevant tickets

References [Pivotal #]().

### Checklist

- [ x ] This PR has been added to the relevant Pivotal card.
- [ x ] Added appropriate feature/unit tests.

### CODE REVIEW

File: cypress/integration/beta-voter-registration-drive-page.js [hunk 1]
```diff
@@ -289,7 +289,7 @@ describe('Beta Voter Registration Drive (OVRD) Page', () => {
 
     cy.get('[data-id=voter-registration-email-field]').type('text@test.com');
     cy.get('[data-id=voter-registration-zip-field]').type('12345');
-    cy.get('[data-id=voter-registration-source-details]').should(
+    cy.get('[data-testid=voter-registration-source-details]').should(
```
Comment (weerd) [inline cypress/integration/beta-voter-registration-drive-page.js:292]: You should be able to change the other `data-id`'s as well right? In the form they're only used for testing purposes?

Comment (Dinnall) [reply cypress/integration/beta-voter-registration-drive-page.js:294]: @weerd Sure thing! Just pushed the changes

**Comment by weerd** [comment]
I realized we don't have the [Testing Library package for Cypress support](https://testing-library.com/docs/cypress-testing-library/intro#__docusaurus) installed and setup yet, so you can't actually use the `getByTestId()` helper just yet 🤦 Buuuut, at least this gets us on the road to renaming our existing data test ids correctly so still is useful!

**Comment by Dinnall** [comment]
> I realized we don't have the [Testing Library package for Cypress support](https://testing-library.com/docs/cypress-testing-library/intro#__docusaurus) installed and setup yet, so you can't actually use the `getByTestId()` helper just yet 🤦 Buuuut, at least this gets us on the road to renaming our existing data test ids correctly so still is useful!

ah, but as you said going forth we can use  `getByTestId()` on other tests once it's installed.
I'm going to merge these changes for now.

**Comment by weerd** [comment]
> ah, but as you said going forth we can use getByTestId() on other tests once it's installed.
I'm going to merge these changes for now.

Yep yep! And you can definitely already use `getByTestId()` in any Jest test 🕺

**Comment by weerd** [comment]
Screen Shot 2020-05-18 at 6 07 40 PM

@Dinnall I was losing my mind trying to figure out why your changes were not showing up on my master when I pulled to get the latest code like 5 times! Turns out you merged this into a different branch 😬 

I'm making the Cypress updates to support Testing Library, so I'll just make these same updates and use the new Cypress helper method in my upcoming PR!

**Pull request merged**